### PR TITLE
LPS-145579 Remove LocalizationUtil usages from modules final part 4

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
@julschong No regressions found.

File: modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/LocalizationImplTest.java [hunk 1]
```diff
@@ -230,42 +229,6 @@ public void testGetLocalizationXmlFromPreferencesWithEmptyPreferences() {
 					"\">testValue</test>"));
 	}
 
-	@Test
-	public void testGetModifiedLocales() throws Exception {
-		String key = RandomTestUtil.randomString();
-
-		String defaultLanguageId = LocaleUtil.toLanguageId(
-			LocaleUtil.getDefault());
```
Comment (tinatian) [inline modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/LocalizationImplTest.java:238]: @julschong Can you move the test to where you inline the code? I do not want to lost test coverage if possible

Tina.

Comment (julschong) [reply modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/LocalizationImplTest.java:238]: @tinatian Hi Tina, as our discussion, this was inlined into archived. we will not be adding the unit test

Comment (tinatian) [reply modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/LocalizationImplTest.java:238]: @julschong Please update comment

**Comment by julschong** [comment]
ci:reopen

**Pull request closed without merging**